### PR TITLE
Bumped Transformation API version

### PR DESCRIPTION
**Pull request by emmberk**

### CODE REVIEW

**Comment by emmberk** [comment]
rebased

build now

**Comment by emmberk** [comment]
I heroed this.

**Comment by emmberk** [comment]
The CI build failed to post the successful status because I accidentally changed permissions. I'm overriding the CI build check on this PR.

File: build.gradle [hunk 1]
```diff
@@ -167,6 +167,7 @@ subprojects {
     repositories {
         mavenLocal()
         mavenCentral()
+        maven { url = uri("http://nexus.phx.connexta.com:8081/nexus/content/repositories/ion-releases/") }
```
Comment (figliold) [inline build.gradle]: ❓ @oconnormi should the above be http://nexus.phx.connexta.com:8081/nexus/content/groups/ion/?

**Pull request merged**